### DISCUSSION TITLE
DockerFile build is Failing

### DISCUSSION THREAD

**Discussion by nov301996** [mlflow/mlflow#6512] [Q&A] [answered]
Hi Guys, The Docker Build is getting failed with the below error, Please have a look at this issue.

8 78.80 Setting up gpg-wks-server (2.2.19-3ubuntu2.2) ...
#8 78.80 Setting up zlib1g-dev:amd64 (1:1.2.11.dfsg-2ubuntu1.3) ...
#8 78.81 Setting up cpp (4:9.3.0-1ubuntu2) ...
#8 78.82 Setting up cmake (3.16.3-1ubuntu1) ...
#8 78.82 Setting up libmysqlclient-dev (8.0.30-0ubuntu0.20.04.2) ...
#8 78.83 Setting up default-libmysqlclient-dev:amd64 (1.0.5ubuntu2) ...
#8 78.83 Setting up gcc-9 (9.4.0-1ubuntu1~20.04.1) ...
#8 78.84 Setting up libstdc++-9-dev:amd64 (9.4.0-1ubuntu1~20.04.1) ...
#8 78.85 Setting up gpg-wks-client (2.2.19-3ubuntu2.2) ...
#8 78.85 Setting up gcc (4:9.3.0-1ubuntu2) ...
#8 78.87 Setting up libprotobuf-dev:amd64 (3.6.1.3-2ubuntu5) ...
#8 78.87 Setting up g++-9 (9.4.0-1ubuntu1~20.04.1) ...
#8 78.89 Setting up g++ (4:9.3.0-1ubuntu2) ...
#8 78.90 update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
#8 78.90 update-alternatives: warning: skip creation of /usr/share/man/man1/c++.1.gz because associated file /usr/share/man/man1/g++.1.gz (of link group c++) doesn't exist
#8 78.91 Setting up gnupg (2.2.19-3ubuntu2.2) ...
#8 78.91 Setting up build-essential (12.8ubuntu1.1) ...
#8 78.92 Processing triggers for libc-bin (2.31-0ubuntu9.9) ...
#8 79.53 Collecting package metadata (current_repodata.json): ...working... failed
#8 86.68 
#8 86.68 CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>
#8 86.68 Elapsed: -
#8 86.68 
#8 86.68 An HTTP error occurred when trying to retrieve this URL.
#8 86.68 HTTP errors are often intermittent, and a simple retry will get you on your way.
#8 86.68 'https//conda.anaconda.org/conda-forge/linux-64'
#8 86.68 
#8 86.68 
------
executor failed running [/bin/sh -c apt-get update &&     apt-get install -y default-libmysqlclient-dev build-essential curl     cmake protobuf-compiler &&      conda install python=3.7 charset-normalizer=2.0.12 &&     pip install -r requirements/dev-requirements.txt --no-cache-dir &&     pip install --no-cache-dir -e . &&     mkdir -p /usr/share/man/man1 && apt-get install -y openjdk-11-jre-headless]: exit code: 1

**Comment by harupy** [comment, marked as answer]
Duplicate of #6513
